doc/netplan.md

```diff
@@ -239,6 +239,12 @@ Virtual devices
 
     Example: ``addresses: [192.168.14.2/24, "2001:1::1/64"]``
 
+``addr-gen-mode`` (scalar)
```

Comment:
All good! Let's just get Steve looking at the key-name - maybe he has something better, seeing that networkd has its own thing for naming it.

Reply:
Which addresses are affected by this?  AIUI it's ipv6-only, but does it affect the behavior of both link-local ipv6 addresses and `accept-ra` ipv6 addresses?

At minimum I think this key should mention ipv6, e.g. `ipv6-address-generation`.